Game of Game of Life.py

```diff
@@ -0,0 +1,110 @@
+### a,b refers to cell (a,b) whereas x,y refers to pixel coordinates ###
+
+import pygame as pg, math as m, time as t, random as r
+
+Width=50    #C 50 How many squares wide the board is
+Height=30   #C 30 Ditto but with height
+Size=20     #C 20 The size of the sides of each square (in pixels)
+Cushion=5   #C 10 How far the board extends beyond the visible amount
+Edge=m.ceil(Size/7)    #C Size/15 The gap between each cell
+Paused=True #Does this need a description?
```

Comment:
No, it doesn't. Instead of making a big comment it's better to make a good variable name.

Reply:
I just made the comments to be sure; they are mostly pretty self-explainitory and it just insures I don't accidently get mixed up. Also, smaller variable names are easier to deal with (eg. I put "Edge" instead of "GapBetweenEachCell" to make it easier to type